lib/build.gradle

```diff
@@ -39,14 +39,23 @@ android {
     }
 }
 
+task createPropertiesFile() {
+    def prop = new Properties()
+    def propFile = new File("${project.projectDir}/src/main/resources/sdk.properties");
+    propFile.createNewFile();
+    prop.setProperty('sdk.version', ANALYTICS_SDK_VERSION)
+    prop.store(propFile.newWriter(), null);
+}
+
```

Comment:
This isn't wrong, but just for consistency, move the task after the dependencies.